src/pages/ReimbursementAccount/CompleteVerification/substeps/ConfirmAgreements.js

```diff
@@ -21,16 +37,38 @@ const validate = (values) => {
         errors.certifyTrueInformation = 'completeVerificationStep.certifyTrueAndAccurateError';
     }
 
-    if (!ValidationUtils.isRequiredFulfilled(values.isControllingOfficer)) {
-        errors.isControllingOfficer = 'completeVerificationStep.isControllingOfficerError';
+    if (!ValidationUtils.isRequiredFulfilled(values.isAuthorizedToUseBankAccount)) {
+        errors.isAuthorizedToUseBankAccount = 'completeVerificationStep.isAuthorizedToUseBankAccountError';
     }
 
     return errors;
 };
 
-function ConfirmAgreements() {
+const COMPLETE_VERIFICATION_KEYS = CONST.BANK_ACCOUNT.COMPLETE_VERIFICATION.INPUT_KEY;
+
+function ConfirmAgreements({onNext, reimbursementAccount}) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
+    const defaultValues = useMemo(
```

Comment:
IMO memo is an overkill here. Its just lodashGet inside of that getDefaultValueForReimbursementAccountField method